batch-setup/provision.sh

```diff
@@ -82,8 +82,8 @@ set -x
 
 python -u /usr/local/src/tileops/import/import.py --find-ip-address meta --date \$DATE \$TILE_ASSET_BUCKET \$AWS_DEFAULT_REGION \
        \$TILE_ASSET_PROFILE_ARN \$DB_PASSWORD
-python -u /usr/local/src/tileops/batch-setup/make_tiles.py --num-db-replicas 10 \$PLANET_DATE --missing-bucket \$MISSING_BUCKET \$RAWR_BUCKET \
-       \$META_BUCKET \$DB_PASSWORD
+python -u /usr/local/src/tileops/batch-setup/make_tiles.py --num-db-replicas 10 \$PLANET_DATE --missing-bucket \$MISSING_BUCKET \
+       --meta-date-prefix \$META_DATE_PREFIX \$RAWR_BUCKET \$META_BUCKET \$DB_PASSWORD
```

Comment:
Will `$META_DATE_PREFIX` always be set? If it isn't, then `--meta-date-prefix` might get the `$RAWR_BUCKET` passed to it and that'd be pretty confusing.

Reply:
Yes, it should always be set in the [`planet-env.sh`](https://github.com/tilezen/tileops/blob/c09ea060478e68e73a67e4286e172e52296ffcdc/batch-setup/provision.sh#L47), which in turn should be [passed in from `cron.py`](https://github.com/tilezen/tileops/blob/c09ea060478e68e73a67e4286e172e52296ffcdc/batch-setup/cron.py#L594), which comes from the command line args or environment, or defaults to the planet date.